Gemfile.lock

```diff
@@ -1,144 +1,200 @@
 GEM
```

Comment:
@liujoshua I'm pretty amateur at fastlane, how does this file get updated?

Reply:
We installed fastlane with bundler. Updated with
`bundle update fastlane`

Reply:
Is that a command you run on your local machine?